docs/docs/guides/integrate/login-ui/passkey.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: Handle Passkey Registration and Login
+sidebar_label: Passkey
+---
+
+Passkeys are a replacement for passwords that provide faster, easier, and more secure sign-ins to websites and apps across a user’s devices.
+It is based on the [FIDO Standard](https://fidoalliance.org/).
+Unlike passwords, passkeys are always strong and phishing-resistant.
+
+## Flow
+
+![Passkey Registration and Login Flow](/img/guides/login-ui/passkey-flow.png)
+
+There are two different possibilities when you like to enable passkey for your user.
+1. You send the user a notification with a code so the user is able to register passkey on another device
+2. You start the passkey registration directly on the device
+
+## Send Registration Link
+
+When you want to send a link to your user, that enables him to register a new passkey device, you can let ZITADEL send the code or return it in the response so you can send a message by yourself.
+
+Make sure to also include the URL Template to customize the reset link in the email sent to the user, in case of ZITADEL sending the email.a
+
+[Send Registration Link Documentation](/apis/resources/user_service/user-service-create-passkey-registration-link)
+
+### Request
+```bash
+curl --request POST \
+  --url https://$ZITADEL_DOMAIN/v2alpha/users/$USER_ID/passkeys/registration_link \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer '"$TOKEN"''\
+  --header 'Content-Type: application/json' \
+  --data '{
+   "sendLink": {
+      "urlTemplate": "https://example.com/passkey/register?userID={{.UserID}}&orgID={{.OrgID}}&codeID={{.CodeID}}&code={{.Code}}"
+   },
+   "returnCode": {}
+}'
+```
+
+## Response in case of return code
+```bash
+{
+	"details": {
+		"sequence": "632",
+		"changeDate": "2023-06-28T08:09:51.257699Z",
+		"resourceOwner": "163840776835432705"
+	},
+	"code": {
+		"id": "220526087715684609",
+		"code": "2KEpIeQGSBBd"
+	}
+}
+```
+
+## Start Passkey Registration
```

Comment:
```suggestion
## Start Passkey registration
```

Reply:
I think we normally do uppercase in the headings.

Reply:
Do we?

I thought we switched to only first character's and names uppercase.

Reply:
At least thats how dakshitha did review my first guide on the login api.
She did everything despite filler words uppercase.
eg. Start the Provider Flow, Register User with Password

But it doesn't matter for me. :)